bindings/py/cpp_src/bindings/engine/py_Engine.cpp

```diff
@@ -457,7 +457,7 @@ namespace htm_ext
                 std::istringstream ss(s);
                 Network self;
                 self.load(ss);
-                return std::move(self);  
+                return self;  
```

Comment:
out of curiosity, why was this the problem?

Reply:
> out of curiosity, why was this the problem?

The `std::move(self) ` code worked fine on MSVC but when I submitted it, the OSx build told me I cannot return a local object that way and it said tor remove the std::move( ).  Since the Network class has a move constructor and no copy constructor it automatically does a move when returning by value.